### ISSUE TITLE
Missing information about git branch in cypress dashboard runs for CodeFresh

### ISSUE THREAD

**Issue by tnrich** [cypress-io/cypress#4609]
Much like similar issues, the cypress dashboard isn't getting the git info from the CI runs in CodeFresh: 

![image](https://user-images.githubusercontent.com/2730609/60373427-fb40c600-99b4-11e9-805b-8608229cd203.png)

Help troubleshooting this would be appreciated! Thanks!

**Comment by jennifer-shehane** [comment]
We determine CI provider and what env vars the CI provider provides here, including how we get any git information: https://github.com/cypress-io/cypress/blob/develop/packages/server/lib/util/ci_provider.js

Here are CodeFresh's exposed environment variables - they need to be added to the ci_provider.js file just like the others and also have tests written within https://github.com/cypress-io/cypress/blob/develop/packages/server/test/unit/ci_provider_spec.coffee#L1

**Comment by bahmutov** [comment]
Example project in https://github.com/bahmutov/cypress-codefresh-example

Codefresh default environment variables https://codefresh.io/docs/docs/codefresh-yaml/variables/#using-codefresh-variables-in-your-pipelines

For example

```
Found environment variables that start with CF_:
CF_ACCOUNT=bahmutov
CF_API_KEY=...
CF_BRANCH=main
CF_BRANCH_TAG_NORMALIZED=main
CF_BRANCH_TAG_NORMALIZED_LOWER_CASE=main
CF_BRANCH_VERSION_NORMALIZED=main
CF_BUILD_ID=602c2258b034d9aa13149d33
CF_BUILD_INITIATOR=bahmutov
CF_BUILD_TIMESTAMP=1613505112419
CF_BUILD_TRIGGER=webhook
CF_BUILD_URL=https://g.codefresh.io/build/602c2258b034d9aa13149d33
CF_CLI_RUN_WAIT_FALLBACK=true
CF_COMMIT_AUTHOR=bahmutov
CF_COMMIT_MESSAGE=print env variables
CF_COMMIT_URL=https://github.com/bahmutov/cypress-codefresh-example/commit/36e18a72ccc87f0d70fe46284ea65c1cc496d69c
CF_CURRENT_ATTEMPT=1
CF_HOST_IP=dind-602c2258b034d9aa13149d33.dind.workflow
CF_HOST_NAME=dind-602c2258b034d9aa13149d33.dind.workflow
CF_MAX_ATTEMPTS=1
CF_PIPELINE_NAME=Cypress Codefresh Example/e2e
CF_PIPELINE_TRIGGER_ID=602c1e1acdbdb25953d416ae
CF_REPO_NAME=cypress-codefresh-example
CF_REPO_OWNER=bahmutov
CF_REVISION=36e18a72ccc87f0d70fe46284ea65c1cc496d69c
CF_SHORT_REVISION=36e18a7
CF_STEP_NAME=test
CF_URL=https://g.codefresh.io
CF_VOLUME_PATH=/codefresh/volume
```

**Comment by jennifer-shehane** [comment]
This was released in [6.6.0](https://on.cypress.io/changelog#6-6-0).
